src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskListTabPanel.tsx

```diff
@@ -109,6 +109,7 @@ const TaskListTabPanel = (props: TaskListTabPanelProps) => {
   const [deadline, setDeadline] = useState(new Date());
   const [priority, setPriority] = useState(TaskPriority.High);
   const [assignees, setAssignees] = useState<UserPreview[]>([]);
+  const [pageSize, setPageSize] = useState(Number(localStorage.getItem('tl-table-row-count')));
```

Comment:
since we use the string `tl-table-row-count` so many times, we should declare it as a constant in all caps at the top of the file like `const TABLE_ROW_COUNT = 'tl-table-row-count';`